packages/analytics/amplify_analytics_pinpoint/example/backend/lib/schema.graphql

```diff
@@ -0,0 +1,22 @@
+type Query {
```

Comment:
This is really great work ... 

Want to make a note here for future work to possibly test the 'updateEndpoint' call being made to Pinpoint as well.  Test should validate that it's called before events are being sent as otherwise Pinpoint won't register those events.  It is the responsibility of the Analytics plugin to automatically call `updateEndpoint` on session start. 

This call will change in the future though as we switch to the new User Centric Model API from Pinpoint.  Regardless, events will need to be associated to either a Pinpoint Endpoint or User object so we'll still want to monitor update calls to Endpoint/User.

Reply:
Nice that's really great!